regcomp.c

```diff
@@ -12730,6 +12730,21 @@ S_regpiece(pTHX_ RExC_state_t *pRExC_state, I32 *flagp, U32 depth)
             }
         }
 
+#if 0				/* Now runtime fix should be reliable. */
+
+        /* if this is reinstated, don't forget to put this back into perldiag:
+
+	    =item Regexp *+ operand could be empty at {#} in regex m/%s/
+
+	   (F) The part of the regexp subject to either the * or + quantifier
+           could match an empty string. The {#} shows in the regular
+           expression about where the problem was discovered.
+
+        */
+
+        if (!(flags&HASWIDTH) && op != '?')
```

Comment:
I didn't touch anything within in #if 0 block, as it would be extra work to try to figure out what was meant, and who knows how much bit rot it had already been subject to.  Realistically, this has been this way for  20 years, lst touched  by Jeffrey Friedl, I very much doubt that tt will get reinstated, and I don't think it's worth trying to figure out what it should do in the nlikely event that we would need to inclde it.  So, I favor leaving it the way it is, or simply removing the whole block

Reply:
I feel we should improve it or remove it, so let's remove it.

Reply:
will do